packages/core/src/Select/SelectButton.tsx

```diff
@@ -0,0 +1,117 @@
+import { forwardRef } from "react";
+import type { Placement } from "@popperjs/core";
+import { DropDownXS } from "@hitachivantara/uikit-react-icons";
+import { theme } from "@hitachivantara/uikit-styles";
+
+import { useDefaultProps } from "../hooks/useDefaultProps";
+import { ExtractNames, createClasses } from "../utils/classes";
+import { HvButton } from "../Button";
+
+export const { staticClasses, useClasses } = createClasses("HvSelectButton", {
+  root: {
+    width: "100%",
+    minWidth: "unset",
+    userSelect: "none",
+    position: "relative",
+    paddingLeft: theme.space.xs,
+    justifyContent: "flex-start",
+  },
+  disabled: {},
+  readOnly: {
+    userSelect: "text",
+    backgroundColor: theme.colors.atmo2,
+  },
+  open: {
+    backgroundColor: theme.colors.atmo1,
+  },
+  openUp: {
+    borderRadius: `0px 0px ${theme.radii.base} ${theme.radii.base}`,
+  },
+  openDown: {
+    borderRadius: `${theme.radii.base} ${theme.radii.base} 0px 0px`,
+  },
+
+  selection: {
+    color: "inherit",
+    flex: 1,
+    textAlign: "start",
+
+    overflow: "clip visible",
+    textOverflow: "ellipsis",
+    whiteSpace: "nowrap",
+  },
+  placeholder: {},
+  arrowContainer: {
```

Comment:
In DS3 the dropdown icon is almost going out of the dropdown:

<img width="326" alt="Screenshot 2024-02-16 at 09 20 34" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/1274a206-6703-40f8-86d7-28ee8e362bb1">

Reply:
Changed styles 👍